graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLWebAutoConfiguration.java

```diff
@@ -104,28 +109,28 @@ public GraphQLSchemaProvider graphQLSchemaProvider(GraphQLSchema schema) {
     @Bean
     @ConditionalOnMissingBean
     public ExecutionStrategyProvider executionStrategyProvider() {
-        if(executionStrategies == null || executionStrategies.isEmpty()) {
+        if (executionStrategies == null || executionStrategies.isEmpty()) {
```

Comment:
Would you mind reverting the formatting changes on these 8 lines?  Otherwise it looks great!

Reply:
Sure, no problem. Will do that right now.